app.properties

```diff
@@ -1,11 +1,11 @@
-project.name:
-project.repo:
-project.sdgs:
+project.name:JaliAjali
+project.repo:https://github.com/BuildForSDG/JaliAjali/
+project.sdgs:"Good Health And Well Being"
 project.app.url:
 project.codacy.url:
 
-project.team.members:
-project.team.slackname:
+project.team.members:"Dominic Ngalo, Steve Ng'etich, Lewis Munyi, Mutugi Mutuma"
```

Comment:
Please change the name to Stephen instead of Steve

Reply:
Fixed.